src/TLuaInterpreter.cpp

```diff
@@ -2361,6 +2361,25 @@ int TLuaInterpreter::disableKey( lua_State *L )
     return 1;
 }
 
+int TLuaInterpreter::killKey( lua_State *L )
+{
+    string luaSendText="";
+    if( ! lua_isstring( L, 1 ) )
+    {
+        lua_pushstring( L, "killKey: wrong argument type" );
```

Comment:
I am not sure that `lua_pushfstring(...)` will work here, `tr("string")` needs `.args()` to supply the replacement parameters `%1`..`%99` - the only other valid replacement string is (I have recently discovered) a `%n` which uses the numeric value as a third argument to `tr("string","context",integer)` to select the right plural/alternative text form to use in conjunction with the actual number positioned where the %n is, some languages have up to three!

E.g. English has "0 rooms", "1 room", "2 rooms"
But French has "0 pièce", "1 pièce", "2 pièces"